app/controllers/track_controller.rb

```diff
@@ -18,4 +18,22 @@ def track
     head :ok
   end
 
+  def decide
+    # from first reply in https://community.mixpanel.com/sending-data-to-mixpanel-11/decide-endpoint-is-throwing-error-500s-on-page-load-1669
+    # decide endpoint of a URL is responsible for providing checks to deliver in-apps and a/b test options to users
+    # not using but called by javascript lib anyway, so set up empty response here with appropriate headers
+    
+    # mixpanel sends withCrednetials so mp_optout is respected but that's not allowed with a *
+    headers['Access-Control-Allow-Origin'] = request.headers['Origin'] || '*'
+    headers['Access-Control-Allow-Credentials'] = true
+
+    headers['Access-Control-Allow-Methods'] = 'GET, POST, PATCH, PUT, DELETE'
+    headers['Access-Control-Allow-Headers'] = 'Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since, X-Requested-With'
+    headers['Access-Control-Max-Age'] = '86400'
+    headers['Cache-Control'] = "no-cache, no-store, max-age=0, must-revalidate"
+    headers['Pragma'] = "no-cache"
+    headers['Expires'] = "Fri, 01 Jan 1990 00:00:00 GMT"
+    head :ok
+  end
+
```

Comment:
This seems more like monkey patching, what do you think about looking for a way to prevent the lib from calling this endpoint to begin with if it's not being used?

Reply:
That was my first approach, to find the part of the client lib calling the endpoint based on the docs, but I couldn't find anything and I also didn't want to inadvertently disable any feature bounty source is currently using that I'm not aware of. I'll go through the docs again and see if I can come up with something else.

Reply:
Okay, I seem to have found something. Based on the docs [here](https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelinit) 'autotrack' is a Mixpanel object config option that enables autotracking on your website[ without any additional implementation.](https://mixpanel.com/de/autotrack/)  A feature that seems like bountysource wasn't using, and it seems like the feature is also being [retired](https://help.mixpanel.com/hc/en-us/articles/115004600343-Autotrack-Retirement) anyway.

 So I have set the `autotrack` config option to false, and the library has stopped calling that endpoint. I browsed bountysource some more locally using a test user and it seems like the current tracking implementation is still working, as the track endpoint is still being called and events are still being logged into the `mixpanel_events` table (mp_page_view, Create Order, etc)